record-and-playback/core/scripts/archive/archive.rb

```diff
@@ -125,20 +125,27 @@ def remux_and_archive(source_dir, dest_dir)
   files .each do |file|
     ext = File.extname(file)
     next if ext.empty?
-    # These can potentially be webm (VP8/VP9), mp4 (H.264), or mkv (VP8/VP9/H.264)
-    output_basename = File.join(dest_dir, File.basename(file, ext))
-    format = {
-      extension: ext.delete_prefix('.'),
-      parameters: [
-        %w[-c copy],
-      ]
-    }
-    BigBlueButton::EDL.encode(
-      nil,
-      file,
-      format,
-      output_basename
-    )
+    begin
```

Comment:
This `begin` / `end` block is actually unnecessary - you can put a `rescue` directly into a `do` / `end` block.

Alternately, to limit the scope of the exception handling, please move the `begin` so that it only catches exceptions raised by the `BigBlueButton::EDL.encode` method.